graphql_compiler/tests/test_helpers.py

```diff
@@ -281,14 +284,35 @@ def get_schema():
     return schema
 
 
+def get_schema_graph():
+    """Get a SchemaGraph object for testing."""
+    return SchemaGraph(get_schema_data())
+
+
+def get_schema_data():
+    """Return the OrientDB schema data for testing."""
+    with open(SCHEMA_DATA_JSON_FILE) as infile:
+        return json.load(infile)
+
+
 def generate_schema(graph_client):
     """Generate schema and type equivalence dict from a pyorient client"""
-    schema_records = graph_client.command(ORIENTDB_SCHEMA_RECORDS_QUERY)
-    schema_data = [x.oRecordData for x in schema_records]
+    schema_graph = generate_schema_graph(graph_client)
     type_overrides = {
         'UniquelyIdentifiable': {'uuid': GraphQLID},
     }
-    return get_graphql_schema_from_orientdb_schema_data(schema_data, type_overrides)
+    return get_graphql_schema_from_schema_graph(schema_graph, type_overrides)
+
+
+def generate_schema_graph(graph_client):
```

Comment:
Make the difference between this function and `get_schema_graph()` painfully clear, in both the name and the docstring. Otherwise, users won't know which one to use / which ones they _should_ use.